resources/codegen/tests/test___main__.py

```diff
@@ -213,8 +213,11 @@ def test_when_none_set_then_all_languages_are_generated(self, tmp_path, yaml_pat
 
 
 def execute_cli(*args):
-    all_args = [sys.executable, "-m", "openassetio_codegen"]
-    all_args.extend(args)
+    # Str wrapping prevents issues with Path objects in Windows
+    # Python 3.7
+    all_args = [str(sys.executable), "-m", "openassetio_codegen"]
```

Comment:
[consider][minor] Pretty sure stringifying `sys.executable` is unnecessary

`sys.executable` [should be a string](https://docs.python.org/3.7/library/sys.html#sys.executable)

> A string giving the absolute path of the executable binary for the Python interpreter

Reply:
Good spot - it seems like it must have been a red-herring before. Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/694/commits/2173016989f7d996d36a9ec79d74f646174fb382.